### PR TITLE
Support total block replica count metric

### LINKED ISSUE #14014: Add TotalBlocks metric as reported by the worker
**Is your feature request related to a problem? Please describe.** 
This is different from UniqueBlock which is from master's point of view. 
We can get a sense of how replicated blocks are from these two metrics.

### PR DESCRIPTION
**Pull request by maobaolong**
### What changes are proposed in this pull request?

Fix #14014

### Does this PR introduce any user facing changes?

Involve a new metric `Master.TotalBlockReplicaCount`

### CODE REVIEW

**Review by yuzhu — COMMENTED** [review]
maybe add a unit test?

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 1]
```diff
@@ -368,6 +368,11 @@ public MetricKey build() {
           .setDescription("Total number of unique blocks in Alluxio")
           .setMetricType(MetricType.GAUGE)
           .build();
+  public static final MetricKey MASTER_TOTAL_BLOCK_REPLICA_COUNT =
+      new Builder("Master.TotalBlockReplicaCount")
```
Comment (apc999) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:372]: I think block replica count should be sufficient
```suggestion
      new Builder("Master.BlockReplicaCount")
```

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 2]
```diff
@@ -368,6 +368,11 @@ public MetricKey build() {
...
+          .setDescription("Total number of block replica count in Alluxio")
```
Comment (apc999) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:373]:
```suggestion
          .setDescription("Total number of block replicas in Alluxio")
```

**Review by apc999 — COMMENTED** [review]
one minor comment modulo david's comments on UT

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**